Algorithms/src/main/java/com/godwije/algos/FastSearch.java

```diff
@@ -0,0 +1,22 @@
+package com.godwije.algos;
+
+import java.util.HashSet;
+import java.util.Set;
+
```

Comment:
Can you please add more comments for your search algo

Reply:
Okay Sure